cognite/src/api/authenticator.rs

```diff
@@ -216,18 +216,23 @@ impl Authenticator {
 
         let status = response.status();
 
+        let response = response.text().await.map_err(|e| {
+            AuthenticatorError::internal_error(
+                "Failed to receive response contents".to_owned(),
```

Comment:
Should we make this more explicitly `to_string()` and not indicating ownership?

```suggestion
                "Failed to receive response contents".to_string(),
```

Reply:
To be honest, I prefer `to_owned` for converting `str` to `string`. `to_string` in my mind is "Convert this type to a string", which is confusing for anyone reading the code not familiar with rust, since this appears to already be a string. `to_owned` is clearer in what it actually does, which is converting a non-owned `str` to an owned `String`.